examples/ConvertPyImageJDataset.ipynb

```diff
@@ -0,0 +1,1153 @@
+       "\t--class io.scif.DefaultImageMetadata--\n",
+       "\t{\n",
+       "\t}\n",
+       "\t\t--class io.scif.AbstractImageMetadata--\n",
+       "\t\t{\n",
+       "\t\tplanarAxes = [net.imagej.axis.DefaultLinearAxis@4a112cb, net.imagej.axis.DefaultLinearAxis@b464d410]\n",
+       "\t\textendedAxes = []\n",
+       "\t\teffectiveAxes = [net.imagej.axis.DefaultLinearAxis@4a112cb, net.imagej.axis.DefaultLinearAxis@b464d410]\n",
+       "\t\tthumbSizeX = 127\n",
+       "\t\tthumbSizeY = 128\n",
+       "\t\tpixelType = 1\n",
+       "\t\tbitsPerPixel = 8\n",
+       "\t\taxes = [net.imagej.axis.DefaultLinearAxis@4a112cb, net.imagej.axis.DefaultLinearAxis@b464d410]\n",
+       "\t\torderCertain = false\n",
+       "\t\tlittleEndian = false\n",
+       "\t\tindexed = false\n",
+       "\t\tplanarAxisCount = 2\n",
+       "\t\tinterleavedAxisCount = -1\n",
+       "\t\tfalseColor = false\n",
+       "\t\tmetadataComplete = false\n",
+       "\t\tthumbnail = false\n",
+       "\t\trois = null\n",
+       "\t\ttables = null\n",
+       "\t\tname = null\n",
+       "\t\ttable = {}\n",
+       "\t\t}\n",
+       "\t\t\t--class java.lang.Object--\n",
+       "\t\t\t{\n",
+       "\t\t\t}\n",
+       "</dd><dt><span>scifio.metadata.global :</span></dt><dd>io.scif.filters.PlaneSeparatorMetadata@3a4aadf8</dd></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.DataArray (row: 408, col: 406)>\n",
+       "array([[160, 185, 208, ..., 192, 184, 166],\n",
+       "       [199, 215, 228, ..., 204, 196, 180],\n",
+       "       [226, 232, 236, ..., 212, 206, 193],\n",
+       "       ...,\n",
+       "       [203, 212, 216, ..., 167, 147, 143],\n",
+       "       [185, 198, 209, ..., 179, 163, 151],\n",
+       "       [156, 172, 189, ..., 191, 180, 164]], dtype=uint8)\n",
+       "Coordinates:\n",
+       "  * row      (row) float64 0.0 1.0 2.0 3.0 4.0 ... 403.0 404.0 405.0 406.0 407.0\n",
+       "  * col      (col) float64 0.0 1.0 2.0 3.0 4.0 ... 401.0 402.0 403.0 404.0 405.0\n",
+       "Attributes:\n",
+       "    rois:                    None\n",
+       "    tables:                  None\n",
+       "    scifio.metadata.image:   io.scif.FieldPrinter@1ca3d25b\\n\\t--class io.scif...\n",
```

Comment:
Ach, yeah, we can do better here. I filed https://github.com/imagej/pyimagej/issues/200.